From 7aecf23ed919012dacf142de7aa1a49d9131570a Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@gmail.com>
Date: Sat, 21 Mar 2015 15:01:39 +0100
Subject: Allow to change include directory.

* On Haiku, includes are not stored in include/, but develop/headers. We
need the buildsystem to be able to adjust itself for this.

diff --git a/makefiles/Makefile.tools b/makefiles/Makefile.tools
index 8a104d7..dd5ff91 100644
--- a/makefiles/Makefile.tools
+++ b/makefiles/Makefile.tools
@@ -22,6 +22,8 @@
 # 				(defaults to /usr/local)
 # LIBDIR		Library installation directory in ${PREFIX}
 # 				(defaults to lib)
+# INCLUDEDIR	Header installtion directory in ${PREFIX}
+# 				(defaults to include)
 #
 
 ###############################################################################
@@ -241,6 +243,9 @@ PREFIX ?= /usr/local
 # Default libdir
 LIBDIR ?= lib
 
+# Default includedir
+INCLUDEDIR ?= include
+
 ###############################################################################
 # Tool defaults
 ###############################################################################
diff --git a/makefiles/Makefile.top b/makefiles/Makefile.top
index 38922d1..476a93f 100644
--- a/makefiles/Makefile.top
+++ b/makefiles/Makefile.top
@@ -292,6 +292,7 @@ define install_pkgconfig
 	$(Q)$(SED) \
 		-e 's#PREFIX#$(PREFIX)#' \
 		-e 's#LIBDIR#$(LIBDIR)#' \
+		-e 's#INCLUDEDIR#$(INCLUDEDIR)#' \
 		-e 's#MAJOR#$(major-version)#' \
 		-e 's#MINOR#$(minor-version)#' \
 		-e 's#PATCH#$(patch-version)#' \
-- 
2.2.2

